.gitignore

```diff
@@ -1,7 +1,6 @@
 .DS_Store
 /node_modules/
 /coverage/
-/dist/
```

Comment:
Hi, sorry for the late review. We should ignore the `dist/` directory as we should only be checking in the source into Git.